src/i18n/languagesProvider.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import LOCALES from './locales';
+
+const LanguagesProvider = ({ children }) => (
```

Comment:
You don't need to create a Context for the languages, it's already in the state ( AppContext)

Reply:
How can i access the languages there ? because i need the lans List to show it in the dropdown, and in the AppContext i can only see the initialState which has the current language stored in the LS

Reply:
AH, hold on, do you mean that i can make use of the AppContext instead of creating a new separated context for the languages ? if so i can jump to that

Reply:
Yes, that's what I meant, and I know you need the list (just import it from ) :)

Reply:
it's already imported (` import LOCALES from './locales' `)

Reply:
Yeah, Alright i made the changes :)